testdata/simple/main.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2016 Google Inc. All rights reserved.
```

Comment:
what conditions are you looking for to check is cgo is set up? e.g. binaries existing in some directory, making sure a binary is on the path, running a command to check the version, etc.